### DISCUSSION TITLE
showAmount returns error with decimal value more than 18

### DISCUSSION THREAD

**Discussion by AKuzmichkin** [CoolBitX-Technology/coolwallet-signing-tools#93] [Q&A] [answered]
- Description
showAmount returns error with decimal value more than 18
Near uses numbers in yoctoNEAR (10^-24 NEAR)

- Expected Behavior
Show correct amount (fix or workaround).

- Screenshots or appendix
![image](https://user-images.githubusercontent.com/101829495/163727013-a257fa48-e17c-4dda-9ff0-32528abe7356.png)

**Comment by skchang0710** [comment, marked as answer]
Hi, there is an update fixing this issue. Please pull master branch and update your card to 324 version and over.
